File: PRAK504-2210817210023-Muhammad Shofy Akmal.py
def reverse (A) :
    C = 0
    while int(A) != 0 :
        sisa = int(A) % 10
        C = C * 10 + sisa
        A = int(A) / 10
    return C
A, B = input().split()
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: PRAK504-2210817210023-MuhammadShofyAkmal.c
#include <stdio.h> 
int reverse(int A){
    int B ; int C = 0 ;
    while (A != 0){
        B = A % 10 ;
        C  = C * 10 + B ;
        A = A / 10 ;}
    return C ;}
int main() { 
int A, B;
scanf("%d %d",&A,&B); 
A=reverse(A); 
B=reverse(B);
int C = A+B; 
printf("%d",reverse(C));
}